static/app/views/performance/transactionSummary/transactionEvents/content.tsx

```diff
@@ -1,214 +1,95 @@
-
-      browserHistory.push({
-        pathname: location.pathname,
-        query: {
-          ...location.query,
-          cursor: undefined,
-          query: searchConditions.formatString(),
-        },
-      });
-    };
-  };
-
-  handleIncompatibleQuery: React.ComponentProps<
-    typeof CreateAlertFromViewButton
-  >['onIncompatibleQuery'] = (incompatibleAlertNoticeFn, _errors) => {
-    const incompatibleAlertNotice = incompatibleAlertNoticeFn(() =>
-      this.setState({incompatibleAlertNotice: null})
-    );
-    this.setState({incompatibleAlertNotice});
-  };
+function EventsContent(props: Props) {
+  const {
+    location,
+    organization,
+    eventView: originalEventView,
+    transactionName,
+    spanOperationBreakdownFilter,
+    webVital,
+    setError,
+  } = props;
 
-  renderError() {
-    const {error} = this.state;
+  const eventView = originalEventView.clone();
 
-    if (!error) {
-      return null;
-    }
+  const transactionsListTitles = [
+    t('event id'),
+    t('user'),
+    t('operation duration'),
+    t('total duration'),
+    t('trace id'),
+    t('timestamp'),
+  ];
 
-    return (
-      <StyledAlert type="error" icon={<IconFlag size="md" />}>
-        {error}
-      </StyledAlert>
-    );
+  if (webVital) {
+    transactionsListTitles.splice(3, 0, t(webVital));
   }
 
-  setError = (error: string | undefined) => {
-    this.setState({error});
-  };
+  const spanOperationBreakdownConditions = filterToSearchConditions(
+    spanOperationBreakdownFilter,
+    location
+  );
 
-  render() {
-    const {eventView, location, organization, projects, transactionName, isLoading} =
-      this.props;
-    const {incompatibleAlertNotice} = this.state;
+  if (spanOperationBreakdownConditions) {
+    eventView.query = `${eventView.query} ${spanOperationBreakdownConditions}`.trim();
+    transactionsListTitles.splice(2, 1, t(`${spanOperationBreakdownFilter} duration`));
+  }
 
-    return (
-      <Fragment>
-        <TransactionHeader
+  return (
+    <Layout.Main fullWidth>
+      <Search {...props} />
+      <StyledTable>
+        <EventsTable
           eventView={eventView}
-          location={location}
           organization={organization}
-          projects={projects}
+          location={location}
+          setError={setError}
+          columnTitles={transactionsListTitles}
           transactionName={transactionName}
-          currentTab={Tab.Events}
-          hasWebVitals="maybe"
-          handleIncompatibleQuery={this.handleIncompatibleQuery}
-        />
-        <Layout.Body>
-          <StyledSdkUpdatesAlert />
-          {this.renderError()}
-          {incompatibleAlertNotice && (
-            <Layout.Main fullWidth>{incompatibleAlertNotice}</Layout.Main>
-          )}
-          <Layout.Main fullWidth>
-            {isLoading ? (
-              <LoadingIndicator />
-            ) : (
-              <Body {...this.props} setError={this.setError} />
```

Comment:
After lifting the header, alerts, and loading states to the parent component, we don't need this component anymore and can just directly use the `Body`